.github/workflows/continuous-integration.yml

```diff
@@ -619,6 +619,10 @@ jobs:
         run: git add .
         working-directory: testing-reports
 
+      - name: Pull any changes since checkout
+        run: git pull origin master
```

Comment:
Does the order matter here? Should this step come before the `git add .`?

Reply:
In my experience it doesn't matter as long as it's before the commit. Happy to move it either way - adding barely takes any time.

Reply:
swapped!